### DISCUSSION TITLE
Auto Assign Issue to current Module.

### DISCUSSION THREAD

**Discussion by mandatoryGithubUser** [makeplane/plane#6276] [Ideas]
When I'm in an issues view, with modules filtered to only `Backend`, creating a new issue will not auto assign that module. 

So for each new issue I need to assign the module, even though the current view is filtered to only select a single module. 

Any chance this could be added? Otherwise how would I go about contributing this?

**Comment by danciaclara** [comment]
What about cases where multiple filters are applied? Assigning based on a single filter may not always work as intended.

We do have a [bulk edit](https://docs.plane.so/core-concepts/issues/bulk-ops) feature that allows you to assign Modules to multiple issues at once. It is a more practical solution for managing assignments efficiently.
